engine/src/main/scala/cromwell/engine/io/IoActor.scala

```diff
@@ -57,19 +61,24 @@ final class IoActor(queueSize: Int,
     val input = builder.add(Flow[IoCommandContext[_]])
     
     // Partitions requests between gcs batch, and single nio requests
-    val batchPartitioner = builder.add(Partition[IoCommandContext[_]](2, {
-      case _: GcsBatchCommandContext[_, _] => 0
-      case _ => 1
+    val batchPartitioner = builder.add(Partition[IoCommandContext[_]](3, {
+      case _: OssCommandContext[_] => 0
```

Comment:
Since you implemented the Nio interface for the OSS filesystem, I don't think any of those modifications are necessary.
The `GcsXXX` classes (except `GcsPath`) are only optimization for GCS. The `DefaultIoCommand`s will use the nio interface to perform the corresponding operations.

Reply:
Ok, these modifies are necessary before an oss nio interface is available, I forget to remove them. I'll solve this problem as soon as possible.